flow/SystemMonitor.cpp

```diff
@@ -298,6 +355,7 @@ SystemStatistics customSystemMonitor(std::string const& eventName, StatisticsSta
 			    .detail("ZoneID", machineState.zoneId)
 			    .detail("MachineID", machineState.machineId)
 			    .trackLatest("MachineMetrics");
+			reportCpuCpuAcctCpuStat(traceEvent);
```

Comment:
will this event info be tracked in fdbcli status?

Reply:
Yes. `status` request will be collecting this event on CC. To report in `status json`, we need to add fields to the json doc.

Reply:
No. Do we want it? @johscheuer